GoogleForCreators/web-stories-wp
# Editor: Fix issue with changing text height due to line breaks

## Summary

This fixes the bug where if a text box has line breaks and use activates edit mode the line breaks were being removed and appearing as if the text box was resizing.

## Relevant Technical Choices

Removed the lookahead regex to correctly replace all linebreaks when importing the content of the text box into DraftJS.


## User-facing changes

Users should no longer see text boxes with linebreaks "resize"/remove the linebreaks when entering into edit mode

## Testing Instructions

1. Create or open an existing story
2. Create a textbox with some dummy text
3. Include a linebreak as the first character and as the last
4. Click off the canvas to exit Edit Mode
5. Double back on the text box to enter Edit Mode
6. See that the text height remains the same and the linebreaks are preserved


### Before
![2218-before](https://user-images.githubusercontent.com/2755722/88850570-92301100-d1b9-11ea-9bb5-193db329492f.gif)

### After
![2218-after](https://user-images.githubusercontent.com/2755722/88850586-9a884c00-d1b9-11ea-98e4-a5d8d2077876.gif)

---



#2218
